test/runBTagAnalyzer_cfg.py

```diff
@@ -473,7 +473,7 @@
         postfix = postfix
     )
     getattr(process,'patJetPartonMatchCA8'+postfix).matched = cms.InputTag(genParticles)
```

Comment:
Actually, this and the following line should be removed since they are automatically handled by the `addJetCollection(...)` function in 74X. Also, CA8 jets should be switched to AK8 jets.